src/structs/field-shorthand.md

```diff
@@ -22,7 +22,6 @@ fn main() {
 }
 ```
 
-<details>
```

Comment:
Hmm, why did we move the `<details>` start tag down? The page becomes very large now.